backend/libs/room-operations.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { DocumentClient } from 'aws-sdk/clients/dynamodb';
+import { Room } from '../../extension/src/types';
+import { createRoom, joinExistingRoom } from './room-operations';
+
+const isTest = process.env.JEST_WORKER_ID;
```

Comment:
Where is `JEST_WORKER_ID` defined ?

Reply:
Jest internals, it was part of their example